src/features/post/components/Post.tsx

```diff
@@ -11,20 +11,25 @@ import {useAuthStore} from "@/features/auth";
 import {
   Comment as CommentType,
   Post as PostType,
+  ReactionStat,
   ReactionType,
 } from "@/services/api/gen";
 import {CommentProvider, PostProvider} from "@/services/api";
 import {useLoading, useToast} from "@/hooks";
 import blankUserProfile from "@/assets/noun-user-picture.svg";
+import {Button} from "@/components/common/button";
 
 export interface PostProps {
   post: PostType;
 }
 
 export const Post: FC<PostProps> = ({post}: PostProps) => {
-  const {queue} = useLoading("reacting_post");
+  const {queue, isLoading} = useLoading("reacting_post");
   const [comments, setComments] = useState<CommentType[]>([]);
   const [isRefresh, setIsRefresh] = useState(false);
+  const [reactions, setReactions] = useState<ReactionStat | undefined>(
```

Comment:
reactions should never be undefined

type should be ReactionStat!

initial value is post.reactions || {likes: 0, dislikes: 0}